### DISCUSSION TITLE
Card Dashboard Help

### DISCUSSION THREAD

**Discussion by txpage** [UI-Lovelace-Minimalist/UI#529] [General]
So I am very new at HA but been learning quite a bit. I followed the instructions and such for minimalist but I keep having this issue with the Minimalist page on the left. Can anyone point me in the right direction?

![image](https://user-images.githubusercontent.com/101153691/158039156-a3ed98c8-3449-42ee-b45a-fbe729323f46.png)

**Comment by Lokalx** [comment]
Also fairly new to HA, and I meticulously followed the instructions and got the same result as you.

**Reply by basbruss** [reply]
Do you also have installed auto-entities through HACS? I think it is still missing in the dependency list

**Reply by Lokalx** [reply]
Thank you for the response.  Yes, I checked the "Include custom card resources" box during setup.  How do I know exactly what is missing?
